apps/modernization-api/src/main/java/gov/cdc/nbs/controller/RedirectController.java

```diff
@@ -19,92 +18,72 @@
-    return redirectionService.handleRedirect(ADVANCED_SEARCH, request, response);
-  }
-  
-  /**
-   * Intercepts patient-profile from legacy page and re-routes/forwards the patient-profile to the modernized 
-   * patient profile page
-   */
-  @ApiIgnore
-  @PostMapping("/nbs/redirect/patientProfile")
-  public RedirectView redirectPatientProfile(
-          @RequestParam (name="MPRUid") String patientId, 
-          HttpServletRequest request,
-          HttpServletResponse response) {
-      return redirectionService.handleRedirect(PATIENT_PROFILE + patientId, request, response);
-  }
-  
-  /**
-   * Sends a GET request to
-   * <WildFly_URL>/nbs/HomePage.do?method=patientSearchSubmit
-   * to set up the session variables so that we can navigate directly to Add Patient or Patient Details pages
-   */
-  @GetMapping("/preparePatientDetails")
-  @ApiImplicitParam(
-            name = "Authorization",
-            required = true,
-            allowEmptyValue = false,
-            paramType = "header",
-            dataTypeClass = String.class)
-  public void preparePatientDetails(HttpServletRequest request) {
-    String url = wildFlyUrl + "/nbs/HomePage.do?method=patientSearchSubmit";
-    // copy cookie header that contains the JSESSIONID from the original request
-    HttpHeaders headers = new HttpHeaders();
-    headers.add("cookie", request.getHeader("cookie"));
-    // send an empty search request to wildfly
-    var emptySearchRequest = RequestEntity.get(url).headers(headers).accept(MediaType.ALL).build();
-    restTemplate.exchange(emptySearchRequest, String.class);
-  }
+    /**
+     * Intercepts legacy advanced search page requests, pulls out the current user from the JSESSIONID, and forwards the
+     * request to the modernization search page
+     */
+    @ApiIgnore
+    @GetMapping("/nbs/redirect/advancedSearch")
+    public RedirectView redirectAdvancedSearch() {
+        return new RedirectView(ADVANCED_SEARCH);
+    }
+
+    /**
+     * Sends a GET request to
+     * <WildFly_URL>/nbs/HomePage.do?method=patientSearchSubmit
+     * to set up the session variables so that we can navigate directly to Add Patient or Patient Details pages
+     */
+    @GetMapping("/preparePatientDetails")
+    @ApiImplicitParam(
+        name = "Authorization",
+        required = true,
+        paramType = "header",
+        dataTypeClass = String.class
+    )
+    public void preparePatientDetails(HttpServletRequest request) {
+        String url = wildFlyUrl + "/nbs/HomePage.do?method=patientSearchSubmit";
+        // copy cookie header that contains the JSESSIONID from the original request
+        HttpHeaders headers = new HttpHeaders();
+        headers.add("cookie", request.getHeader("cookie"));
+        // send an empty search request to wildfly
+        var emptySearchRequest = RequestEntity.get(url).headers(headers).accept(MediaType.ALL).build();
+        restTemplate.exchange(emptySearchRequest, String.class);
+    }
```

Comment:
Is this still being used anywhere? Since we are building out the `Add Patient` and `Patient Details` pages, we might be able to remove this.

Reply:
There are a few places in the client that are using this endpoint still.  It will be phased out after [CNFT1-853](https://cdc-nbs.atlassian.net/browse/CNFT1-853) and [CNFT1-854](https://cdc-nbs.atlassian.net/browse/CNFT1-854).  There will be PR's for these stories soon, the latter will remove `/preparePatientDetails`